static/api/v2.yaml

```diff
@@ -1581,30 +1581,710 @@ paths:
+        write a description about passkeys
+      security:
+        - BearerAuth:
+            - 'passkeys:revoke'
+      parameters:
+        - $ref: '#/components/parameters/passkey_id'
+      responses:
+        '201':
+          description: |
+            No response body needed as this is just a revocation.
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Invalid Parameters:
+                  $ref: '#/paths/~1v2~1users/get/responses/400/content/application~1json/examples/Invalid%20Parameters'
+        '401':
+          description: Unauthorized.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Missing Authorization:
+                  $ref: '#/paths/~1v2~1users/get/responses/401/content/application~1json/examples/Missing%20Authorization'
+        '403':
+          description: Forbidden.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Insufficient Authorization:
+                  $ref: '#/paths/~1v2~1users/get/responses/403/content/application~1json/examples/Insufficient%20Authorization'
+        '404':
+          description: Not found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                User Not Found:
+                  value:
+                    code: not_found
+                    message: user not found
+                    details:
+                      - type: ResourceInfo
+                        resource_type: User
+                        id: 37660157-aa4b-4e84-958f-1fdf27777e86
+                        description: user not found
+        '412':
+          description: Precondition Failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                User Not Active:
+                  value:
+                    code: failed_precondition
+                    message: user not active
+                    details:
+                      - type: ResourceInfo
+                        resource_type: User
+                        id: 37660157-aa4b-4e84-958f-1fdf27777e86
+                        description: user not active
+        '500':
+          description: Server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Internal Error:
+                  $ref: '#/paths/~1v2~1users/get/responses/500/content/application~1json/examples/Internal%20Error'
+  /v2/passkeys/tags:
```

Comment:
This isn't part of the implementation. The tags are returned as part of the list passkeys endpoint.